File: tests/cs/hello.cs
/*
 * tests/hello.cs
 * Minimal C# test — requires dotnet SDK installed.
 * Run with: crun tests/hello.cs
 * Note: crun will generate a temporary .csproj if none exists.
 */
using System;

class Hello {
    static void Main() {
        Console.WriteLine("crun: C# compilation OK");
    }
}

File: tests/objc/hello.m
/*
 * tests/hello.m
 * Minimal Objective-C test — uses only the ObjC runtime, no Foundation/AppKit.
 * Avoids GNUstep dependency so it works on any Linux with clang + libobjc.
 * Run with: crun tests/hello.m
 */
#include <stdio.h>

int main(void) {
    printf("crun: Objective-C compilation OK\n");
    return 0;
}